enterprise_access/apps/bffs/tests/test_context.py

```diff
@@ -0,0 +1,68 @@
+"""
+Text for the BFF context
+"""
+from django.test import RequestFactory, TestCase
+
+from enterprise_access.apps.api_client.tests.test_constants import DATE_FORMAT_ISO_8601
+from enterprise_access.apps.bffs.context import HandlerContext
+from enterprise_access.apps.core.tests.factories import UserFactory
+from enterprise_access.utils import _curr_date
+
+
+class TestHandlerContext(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.factory = RequestFactory()
+        self.mock_user = UserFactory()
+        self.mock_error_user_message = "No associated enterprise customer uuid"
+        self.mock_error_developer_message = "Unable to complete request, created: {}, X_Request_ID: {}".format(
+            _curr_date(DATE_FORMAT_ISO_8601),
+            "12345"
+        )
+
+    def test_handler_context_init(self):
+        request = self.factory.get('sample/api/call')
+        request.user = self.mock_user
+        context = HandlerContext(request)
+
+        self.assertEqual(context.request, request)
+        self.assertEqual(context.user, self.mock_user)
+        self.assertEqual(context.data, {})
+        self.assertEqual(context.errors, [])
+        self.assertEqual(context.warnings, [])
+        self.assertEqual(context.enterprise_customer_uuid, None)
+        self.assertEqual(context.lms_user_id, None)
+
+    def test_handler_context_add_error(self):
+        request = self.factory.get('sample/api/call')
+        request.user = self.mock_user
+        context = HandlerContext(request)
+        expected_output = {
+            "developer_message": "No enterprise uuid associated to the user mock-uuid",
+            "user_message": "You may not be associated with the enterprise.",
+        }
+        arguments = {
+            **expected_output,
+            "status": 403
```

Comment:
[clarification] Does the `add_error` / `add_warning` methods accept/use `status` kwarg?

Reply:
It does not but its purpose is to specifically test that the serializer is correctly serializing only the relevant attributes. This also allows the output to be purely serializer driven.

Reply:
Cool, makes sense. Just for posterity, might recommend just leaving a comment next to `status` to explain its rationale and intent to mitigate any confusion around why its included but not explicitly used.